lib/Resizable.js

```diff
@@ -234,25 +274,17 @@ export default class Resizable extends React.Component<Props, State> {
   }
 
   render(): ReactNode {
-    // eslint-disable-next-line no-unused-vars
-    const {children, draggableOpts, width, height, handleSize,
-        lockAspectRatio, axis, minConstraints, maxConstraints, onResize,
-        onResizeStop, onResizeStart, resizeHandles, transformScale, ...p} = this.props;
-
-    const className = p.className ?
-      `${p.className} react-resizable`:
-      'react-resizable';
+    const {children, draggableOpts, resizeHandles, className} = this.props;
```

Comment:
Just double-checked, and this won't cause any breakage in React-Grid-Layout :+1:

https://github.com/STRML/react-grid-layout/blob/83251e5e682abfa3252ff89d4bacf47fdc1f4270/lib/GridItem.jsx#L351-L368

Reply:
This part is the core of my fix.
Using your example `<Resizable foo="bar"><div /></Resizable>` we would find in dev console an error saying that we passed a prop/s not recognized by the div component.

The component would look like this `<div foo="bar" />`. For this reason the `handle error` appeared. This cloning of props is not a behavior described in the documentation. For this reason we should decide whether we want to support it or not. I would not recommend it because this could lead to a large amount of issue reports on the library

Reply:
I am thinking to revert this change, as it actually does cause breakage in React-Grid-Layout. In this case we will abandon the 2.x branch until an actual breaking change is created.

The reasoning is pretty simple: imagine this tree:

```jsx
<DraggableCore ...>
  <Resizable ...>
    <div />
  <Resizable />
<DraggableCore />
```

The `<DraggableCore>` will not be able to attach event listeners to the underlying `<div />`, because it is assuming that it can attach them to its child. Short of creating an explicit carve-out for detecting `Resizable`, which I don't believe is wise, I think it makes sense to again start forwarding props.